platform/core/src/types/HangingProtocol.ts

```diff
@@ -293,6 +293,16 @@ export type Protocol = {
    */
   numberOfPriorsReferenced?: number;
   syncDataForViewports?: boolean;
+  /* Set of minimal conditions necessary to run the hanging protocol
+   */
+  hpInitiationCriteria?: {
+    /* If configured, sets the minimum number of series needed to run the hanging
+     * protocol and start displaying images. Used when OHIF needs to handle studies
+     * with several series and it is required that the first image should be loaded
+     * faster.
+     */
+    minSeriesLoaded: number;
+  };
```

Comment:
I think this also makes sense to be a configuration as in configuration file, what do you think?

Reply:
No, I think it has to be in the hanging protocol - it is really about what the hanging protocol needs, and eventually it will be important to know how much data is needed for all of the potentially applicable hanging protocols before starting.